src/tests/ftest/server/replay.py

```diff
@@ -0,0 +1,66 @@
+"""
+  (C) Copyright 2019-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from apricot import TestWithServers
+from ior_utils import run_ior
+from job_manager_utils import get_job_manager
+from test_utils_pool import add_pool
+
+
+class ReplayTests(TestWithServers):
+    """Shutdown/restart/replay test cases.
+
+    Restarting engines with volatile SCM will include loading the blob from the SSD and re-applying
+    any changes from the WAL.
+
+    :avocado: recursive
+    """
+
+    def test_restart(self):
```

Comment:
Similar to previous comment, how do we ensure these are unique? E.g. maybe?
```suggestion
    def test_server_replay_restart(self):
```